arrows/darknet/darknet_detector.cxx

```diff
@@ -275,15 +276,15 @@ check_configuration( vital::config_block_sptr config ) const
 } // darknet_detector::check_configuration
 
 
-// --------------------------------------------------------------------
+// -----------------------------------------------------------------------------
 vital::detected_object_set_sptr
 darknet_detector::
 detect( vital::image_container_sptr image_data ) const
 {
   kwiver::vital::scoped_cpu_timer t( "Time to Detect Objects" );
 
-  cv::Mat cv_image = kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
-  cv::Mat cv_resized_image;
+  cv::Mat cv_resized_image, cv_image =
+    kwiver::arrows::ocv::image_container::vital_to_ocv( image_data->get_image() );
 
```

Comment:
I agree.  This struck me as odd.  I liked to old way better.  It was more clear.  This way is more likely to confuse a C++ novice.  One might assume that you are trying to assign the result of the function to both variables.